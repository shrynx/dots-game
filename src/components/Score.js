// @flow
import styled from 'styled-components';
import { PURPLE } from '../styles/colors';

const Score = styled.div`
  font-size: 32px;
  font-weight: 600;
  text-align: center;
  color: ${PURPLE};
  margin-bottom: 10px;
`;

export default Score;
