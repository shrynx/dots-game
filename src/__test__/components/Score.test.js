import React from 'react';
import { render } from 'react-testing-library';
import Score from '../../components/Score';

it('Score component should render correctly', () => {
  const { container } = render(<Score>200</Score>);
  expect(container).toMatchSnapshot();
});
