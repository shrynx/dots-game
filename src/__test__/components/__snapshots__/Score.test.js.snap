// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Score component should render correctly 1`] = `
<div>
  <div
    class="sc-bdVaJa ihkZqg"
  >
    200
  </div>
</div>
`;
